springwolf-add-ons/springwolf-kotlinx-serialization-model-converter/src/main/java/io/github/springwolf/addons/kotlinx_serialization_model_converter/configuration/KotlinxSerializationModelConverterAutoConfiguration.java

```diff
@@ -2,19 +2,24 @@
 package io.github.springwolf.addons.kotlinx_serialization_model_converter.configuration;
 
 import io.github.springwolf.addons.kotlinx_serialization_model_converter.converter.KotlinxSerializationModelConverter;
+import io.github.springwolf.core.configuration.properties.SpringwolfConfigConstants;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import static io.github.springwolf.core.configuration.properties.SpringwolfConfigConstants.SPRINGWOLF_USE_FQN;
+
 /**
  * Spring AutoConfiguration adding an {@link KotlinxSerializationModelConverter} Bean to the spring context.
  */
 @Configuration(proxyBeanMethods = false)
+@ConditionalOnProperty(name = SpringwolfConfigConstants.SPRINGWOLF_ENABLED, havingValue = "true", matchIfMissing = true)
 public class KotlinxSerializationModelConverterAutoConfiguration {
 
     @Bean
     public KotlinxSerializationModelConverter kotlinxSerializationTypeConverter(
-            @Value("${springwolf.use-fqn}") boolean useFqn) {
+            @Value("${" + SPRINGWOLF_USE_FQN + ":true}") boolean useFqn) {
```

Comment:
Are you able to inject the complete Springwolf ConfigDocket, which has the useFqn property? 
In contains the correct default value as well